backend/src/org/commcare/suite/model/graph/Graph.java

```diff
@@ -173,6 +185,27 @@ private void evaluateSeries(GraphData graphData, EvaluationContext context) {
                     }
                 }
                 graphData.addSeries(seriesData);
+
+                for (Enumeration<String> e = expandedConfiguration.keys(); e.hasMoreElements();) {
+                    String key = e.nextElement();
+                    String json = "";
+                    for (String pointValue : expandedConfiguration.get(key)) {
+                        json += "," + "'" + pointValue.replaceAll("'", "&apos;") + "'";
+                    }
+                    json = "[" + json.substring(1) + "]";
+                    Text value = Text.PlainText(json);
+                    s.setConfiguration(key, value);
+                }
+
+                // Handle configuration after data, since data processing may update configuration
```

Comment:
Alternatively, the expanded configuration could be stored in a new configuration key, in case the Graph gets evaluated multiple times. Seemed slightly cleaner to have fewer keys and to have the XPath function gone by the time cc-odk gets the Graph.